app/src/pages/inside/stepPage/stepGrid/defectType/issueList/issue/issueInfoTooltip/issueInfoTooltip.jsx

```diff
@@ -59,6 +60,7 @@ const FETCH_ISSUE_INTERVAL = 900000; // min request interval = 15 min
 
 @connect((state, ownProps) => ({
   activeProject: activeProjectSelector(state),
```

Comment:
Do we need it?

Reply:
I think yes. As I understand we use activeProject in tooltip.